sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -852,6 +852,11 @@ public partial class BlobTag
         public string Key { get { throw null; } set { } }
         public string Value { get { throw null; } set { } }
     }
+    public partial class BlobTagRequestConditions : Azure.RequestConditions
+    {
+        public BlobTagRequestConditions() { }
+        public string TagConditions { get { throw null; } set { } }
```

Comment:
What's the format of this string?

Reply:
Something along the lines of `"coolTag" = 'true'`.

For more information, see the "Tags Conditional Operations" section of [Blob Tags Spec](https://microsoft.sharepoint.com/:w:/r/teams/AzureStorage/_layouts/15/Doc.aspx?sourcedoc=%7B37E4358D-61E9-4AEB-B1E7-156636A48C61%7D&file=Tags%20Conditional%20Operations.docx&action=default&mobileredirect=true&cid=4f62c1cf-9c5b-4748-af90-704d320e801a).

We have an email thread a few months ago with the Tags team and your team about providing more support for this syntax in the SDK, but decided against it.

Reply:
Thanks. Let's just point people to good docs.